pyshp/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-$PYTHON setup.py install
+$PYTHON setup.py install install --single-version-externally-managed --record record.txt
```

Comment:
I think we might be able to start pulling these one liners (which are shared between all platforms) into the meta.yaml.
What do you think?